_pages/cheri-feature-matrix.md

```diff
@@ -133,17 +133,17 @@ flow:
             | Kernel temporal safety | - | - |
             | KGDB support for memory-safe kernels | Yes | -[^1] |
             | KGDB support for memory-unsafe kernels | Yes | - (as above) |
-            | Userspace spatial safety<br />(w/memory-unsafe kernel) | Yes | Yes
+            | Userspace spatial safety<br />(w/memory-unsafe kernel) | Yes | Yes |
             | Userspace spatial safety<br />(w/memory-safe kernel) | Yes | - |
-            | Userspace temporal safety | Experimental<br />(branch - expected 23.06 release) | - |
+            | Userspace temporal safety | Experimental<br />([branch](https://github.com/CTSRD-CHERI/cheripedia/wiki/HOWTO:-Use-Cornucopia-with-the-22.12-CheriBSD-Release) - expected 23.06 release) | - |
             | Debugger for memory-safe userspace | Yes | Yes |
             | OS tracing for memory-unsafe userspace<br />(ftrace, DTrace, eBPF, ...) | Under development | Yes |
             | OS tracing for memory-safe userspace<br />(ftrace, DTrace, eBPF, ...) | - | - |
-            | CHERI-enabled Type-2 hypervisor<br />(kvm, bhyve, ...) | Experimental<br />(branch - expected 23.06 release) | Under development |
+            | CHERI-enabled Type-2 hypervisor<br />(kvm, bhyve, ...) | Experimental<br />([branch](https://github.com/CTSRD-CHERI/cheribsd/tree/morello-bhyve) - expected 23.06 release) | Under development |
             | Userspace memory-safety protection<br />against kernel confused deputies | Yes | Under development |
-            | Kernel module compartmentalisation | Experimental<br />(branch) | - |
-            | Userspace library compartmentalisation | Experimental<br />(shipping) | - |
-            | Userspace co-process compartmentalisation | Experimental<br />(branch) | - |
+            | Kernel module compartmentalisation | Experimental<br />([branch](https://github.com/CTSRD-CHERI/cheribsd/tree/kernel-compart-linker)) | - |
+            | Userspace library compartmentalisation | Experimental<br />([shipping](https://github.com/CTSRD-CHERI/cheripedia/wiki/Library-based-Compartmentalisation)) | - |
```

Comment:
This might want to be a link to https://man.cheribsd.org/cgi-bin/man.cgi/c18n instead.